components/shared/Blocks/Features.tsx

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+import Actions from "./ActionsButton";
+import { tinaField } from "tinacms/dist/react";
+import Image from "next/image";
+import { Components, TinaMarkdown, TinaMarkdownContent } from "tinacms/dist/rich-text";
+import { YouTubeEmbed } from "../YouTubeEmbed";
+import { ButtonSize, ButtonVariant } from "./buttonEnum";
+
+
+type ActionButton = {
+  label: string;
+  url: string;
+  variant: ButtonVariant
+  size: ButtonSize
+};
+
+export type FeatureItem = {
+  headline: string;
+  text: TinaMarkdownContent;
+  buttons: ActionButton[];
+  media: Array<{
+    __typename: string;
+    image?: string;
+    src?: string;
+  }>;
+  isReversed: boolean;
+  hasBackground: boolean;
+};
+
+
+export const featureComponents: Components<Record<string, unknown>> = { 
+  //@ts-expect-error investigate type err
+  p: (props: React.HTMLProps<HTMLParagraphElement>) => (
+    <p
+      className="mb-4 text-white font-helvetica leading-relaxed lg:text-lg md:text-md"
+      //@ts-expect-error investigate type err
+      data-tina-field={tinaField(props, 'text')}
+      {...props}
+    />
+  ),
+  //@ts-expect-error investigate type err
+  span: (props: React.HTMLProps<HTMLSpanElement>) => (
+    <span
+      className="mb-4 text-white font-helvetica leading-relaxed lg:text-lg md:text-md"
+      //@ts-expect-error investigate type err
+      data-tina-field={tinaField(props, 'text')}
+      {...props}
+    />
+  ),
```

Comment:
`//@ts-expect-error investigate type err`

It's rare these are actually required, make a PBI to investigate later

Reply:
https://github.com/SSWConsulting/SSW.Products/issues/18